src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import Accordion from './Accordion';
+import AccordionDetail from './AccordionDetail';
+import AccordionHeader from './AccordionHeader';
+
+export default {
+  title: 'Accordion',
+  component: Accordion,
+};
+
+export const DefaultAccordion = () => {
+  return  <Accordion  onClick={action('hello clicked')}>
+            <AccordionHeader >
+              <p>hello</p>
+              <p>hello</p>
+              <p>hello</p>
+              <p>asdasdfasdfsadfsadfsadfsadfasdfsadfsadfsadfsadffsadfsadfsadfsadf</p>
+            </AccordionHeader>
+            <AccordionDetail>
+              <p>New Accoirdon detail helloooo</p>
+            </AccordionDetail>
+          </Accordion>
+}
+
+export const MultipleAccordions = () => {
+  return  <><Accordion>
+            <AccordionHeader>
+              <p>hello</p>
+            </AccordionHeader>
+            <AccordionDetail>
+              <p>New Accoirdon detail helloooo</p>
+            </AccordionDetail>
+          </Accordion>
+          <Accordion>
+          <AccordionHeader>
+            <p>hello</p>
+          </AccordionHeader>
+          <AccordionDetail>
+            <p>New Accoirdon detail helloooo</p>
+          </AccordionDetail>
+        </Accordion>
+        <Accordion>
+        <AccordionHeader>
+          <p>hello</p>
+        </AccordionHeader>
+        <AccordionDetail>
+          <p>New Accoirdon detail helloooo</p>
+        </AccordionDetail>
+      </Accordion>
+      </>
+}
+
+
+export const DisabledAccordion = () => {
+  return  <><Accordion>
+            <AccordionHeader>
+              <p>hello</p>
+            </AccordionHeader>
+            <AccordionDetail>
+              <p>New Accoirdon detail helloooo</p>
+            </AccordionDetail>
+          </Accordion>
+          <Accordion>
+          <AccordionHeader>
+            <p>hello</p>
+          </AccordionHeader>
+          <AccordionDetail>
+            <p>New Accoirdon detail helloooo</p>
+          </AccordionDetail>
+        </Accordion>
+        <Accordion disabled>
+        <AccordionHeader>
+          <p>hello</p>
+        </AccordionHeader>
+        <AccordionDetail>
+          <p>New Accoirdon detail helloooo</p>
+        </AccordionDetail>
+      </Accordion>
+      </>
+}
+
+export const ControlledAccordion:React.FC = () => {
+  const [expanded, setExpanded] = React.useState<string | false >(false);
+
+  const handleChange = (panel : string) => {
+    return (open?: boolean) => setExpanded(open ? panel : false)
```

Comment:
-using an empty string instead of False should provide the same result based on this logic. this will also allow you to remove the false type since that may be considered too specific paired with a string data type

Reply:
fixed bad logic